code/modules/mapfluff/ruins/icemoonruin_code/wrath.dm

```diff
@@ -10,22 +10,4 @@
 
 /obj/item/clothing/gloves/butchering/Initialize(mapload)
 	. = ..()
-	AddComponent(/datum/component/butchering, 5, 125, null, null, TRUE, TRUE)
-
-/obj/item/clothing/gloves/butchering/equipped(mob/user, slot, initial = FALSE)
-	. = ..()
-	RegisterSignal(user, COMSIG_HUMAN_EARLY_UNARMED_ATTACK, .proc/butcher_target)
-	var/datum/component/butchering/butchering = src.GetComponent(/datum/component/butchering)
-	butchering.butchering_enabled = TRUE
-
-/obj/item/clothing/gloves/butchering/dropped(mob/user, silent = FALSE)
-	. = ..()
-	UnregisterSignal(user, COMSIG_HUMAN_EARLY_UNARMED_ATTACK)
-	var/datum/component/butchering/butchering = src.GetComponent(/datum/component/butchering)
-	butchering.butchering_enabled = FALSE
-
-/obj/item/clothing/gloves/butchering/proc/butcher_target(mob/user, atom/target, proximity)
-	SIGNAL_HANDLER
-	if(!isliving(target))
-		return
-	return SEND_SIGNAL(src, COMSIG_ITEM_ATTACK, target, user)
+	AddComponent(/datum/component/butchering/wearable, 5, 125, null, null, TRUE, TRUE)
```

Comment:
what do all these numbers and such mean? can you do something like this instead
```suggestion
	AddComponent(/datum/component/butchering/wearable, _speed = 5, _effectiveness = 125, null, null, TRUE, TRUE)
```
for all of them

Reply:
true I didnt notice it at all

Reply:
alright made all of datum component butchering up to modern standards